src/shadowbox/README.md

```diff
@@ -1,203 +1,169 @@
+   - **Deployed with the Outline Manager:** Check the "Settings" tab.
 
-```
-API_URL=https://1.2.3.4:1234/3pQ4jf6qSr5WVeMO0XOo4z
-```
+   - **Local Deployments from Source:** `https://[::]:8081/TestApiPrefix`
 
-You can then perform the following operations on the server, remotely.
+1. **API Examples:**
 
-List access keys
+   Replace `$API_URL` with your actual `apiUrl`.
 
-```
-curl --insecure $API_URL/access-keys/
-```
+   - **List access keys:** `curl --insecure $API_URL/access-keys/`
 
-Create an access key
+   - **Create an access key:** `curl --insecure -X POST $API_URL/access-keys`
 
-```
-curl --insecure -X POST $API_URL/access-keys
-```
+   - **Get an access key (e.g. ID 1):** `curl --insecure $API_URL/access-keys/1`
 
-Get an access key (e.g. get access key 1)
+   - **Rename an access key:** `curl --insecure -X PUT -F 'name=albion' $API_URL/access-keys/2/name`
 
-```
-curl --insecure $API_URL/access-keys/1
-```
+   - **And more...**
 
-Rename an access key
-(e.g. rename access key 2 to 'albion')
+1. **Further Options:**
 
-```
-curl --insecure -X PUT -F 'name=albion' $API_URL/access-keys/2/name
-```
+   Consult the [OpenAPI spec](./server/api.yml) and [documentation](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/Jigsaw-Code/outline-server/master/src/shadowbox/server/api.yml) for more options.
 
-Remove an access key
-(e.g. remove access key 2)
-
-```
-curl --insecure -X DELETE $API_URL/access-keys/2
-```
+## Testing
 
-Set a data limit for all access keys
-(e.g. limit outbound data transfer access keys to 1MB over 30 days)
+### Manual
 
-```
-curl -v --insecure -X PUT -H "Content-Type: application/json" -d '{"limit": {"bytes": 1000}}' $API_URL/experimental/access-key-data-limit
-```
+1. Prerequisites
 
-Remove the access key data limit
+   - A locally built Docker image containing your modifications.
+   - Your Docker image uploaded to a registry (e.g., Docker Hub, Quay.io).
 
-```
-curl -v --insecure -X DELETE $API_URL/experimental/access-key-data-limit
-```
+1. Setup
 
-## Testing
+   - **Environment Variable:** Set `SB_IMAGE` to the uploaded image location. Example:
 
-### Manual
+     ```sh
+     export SB_IMAGE=yourdockerhubusername/shadowbox
+     ```
 
-After building a docker image with some local changes,
-upload it to your favorite registry
-(e.g. Docker Hub, quay.io, etc.).
+   - **Start the Server:**
 
-Then set your `SB_IMAGE` environment variable to point to the image you just
-uploaded (e.g. `export SB_IMAGE=yourdockerhubusername/shadowbox`) and
-run `npm run action server_manager/electron_app/start` and your droplet should be created with your
-modified image.
+     ```sh
+     npm run action shadowbox/docker/start
+     ```
 
 ### Automated
 
-To run the integration test:
-
-```
-npm run action shadowbox/integration_test/run
-```
+1. **Build and Run:**
 
-This will set up three containers and two networks:
+   ```sh
+   npm run action shadowbox/integration_test/run
```

Comment:
I think you need to pass the image name?

Reply:
I think that's only the case for the `test.sh` file, at least according to https://github.com/Jigsaw-Code/outline-server/blob/71e4b52b80abfa2fac36eda98b64763452b92102/src/shadowbox/integration_test/README.md. I think `run.action.sh` looks at `SB_IMAGE` only.